tests/sonic_xcvr/test_xcvr_field.py

```diff
@@ -0,0 +1,208 @@
+from sonic_platform_base.sonic_xcvr.fields.xcvr_field import (
+    CodeRegField,
+    HexRegField,
+    NumberRegField,
+    RegBitField,
```

Comment:
There's not much point, this is covered by NumberRegField and CodeRegField. In practice RegField would not actually be declared in XcvrMemMaps directly since its decode() is abstract.